sockeye/average.py

```diff
@@ -166,7 +169,12 @@ def main():
         help="either a single model directory (automatic checkpoint selection) "
              "or multiple .params files (manual checkpoint selection)")
     params.add_argument(
-        "--max", action="store_true", help="maximize metric (default: min)")
+        "--max", action="store_true", help="Maximize metric.")
+    params.add_argument(
+        "--metric",
+        help="Name of the metric to choose n-best checkpoints from. (default: {})".format(C.PERPLEXITY),
```

Comment:
in the help message you can write : "Default: %(default)s" to automatically take over what is set as default in kwargs (see arguments.py)